### PR TITLE
Rename meilisearch_host to meilisearch_url

### PR DESCRIPTION
**Pull request by AdoPi**
Hello :wave: 

I think the name meilisearch_host is a little bit confusing and error prone as it is used for an URL and not a real hostname.

I've changed the name "host" to use "url" instead.

## PR checklist
Please check if your PR fulfills the following requirements:
- [ X] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [X ] Have you read the contributing guidelines?
- [X ] Have you made sure that the title is accurate and descriptive of the changes?

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ services:
     stdin_open: true
     working_dir: /home/package
     environment:
-      - MEILISEARCH_HOST=http://meilisearch:7700
+      - MEILISEARCH_URL=http://meilisearch:7700
```
Comment (brunoocasali) [inline docker-compose.yml:13]: Can you change to MEILISEARCH_HOST again?
```suggestion
      - MEILISEARCH_HOST=http://meilisearch:7700
```

All the other SDKs, share the same principles, and if I change here I will have to change in every other repo, I agree with your suggestion and I will create an issue for that :)

But for now to merge your PR I have to ask you to change the docker config back ;)

**Review by brunoocasali — CHANGES_REQUESTED** [review]
Hello @AdoPi thanks a lot for your contribution!

I just asked for one fix, after that it will be good to go!

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -45,7 +45,7 @@
   # Remove all indexes setup in this run in local or CI
   c.after(:suite) do
     MeiliSearch::Rails.configuration = {
-      meilisearch_host: ENV.fetch('MEILISEARCH_HOST', 'http://127.0.0.1:7700'),
+      meilisearch_url: ENV.fetch('MEILISEARCH_URL', 'http://127.0.0.1:7700'),
```
Comment (brunoocasali) [inline spec/spec_helper.rb:48]:
```suggestion
      meilisearch_url: ENV.fetch('MEILISEARCH_HOST', 'http://127.0.0.1:7700'),
```

File: spec/support/1_initialize_meilisearch.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 MeiliSearch::Rails.configuration = {
-  meilisearch_host: ENV.fetch('MEILISEARCH_HOST', 'http://127.0.0.1:7700'),
+  meilisearch_url: ENV.fetch('MEILISEARCH_URL', 'http://127.0.0.1:7700'),
```
Comment (brunoocasali) [inline spec/support/1_initialize_meilisearch.rb:2]:
```suggestion
  meilisearch_url: ENV.fetch('MEILISEARCH_HOST', 'http://127.0.0.1:7700'),
```

File: spec/integration_spec.rb [hunk 4]
```diff
@@ -687,7 +687,7 @@
 
 describe 'attributes_to_crop' do
   before(:all) do
-    MeiliSearch::Rails.configuration = { meilisearch_host: ENV.fetch('MEILISEARCH_HOST', 'http://127.0.0.1:7700'),
+    MeiliSearch::Rails.configuration = { meilisearch_url: ENV.fetch('MEILISEARCH_URL', 'http://127.0.0.1:7700'),
```
Comment (brunoocasali) [inline spec/integration_spec.rb:690]:
```suggestion
    MeiliSearch::Rails.configuration = { meilisearch_url: ENV.fetch('MEILISEARCH_HOST', 'http://127.0.0.1:7700'),
```

File: spec/integration_spec.rb [hunk 5]
```diff
@@ -646,7 +646,7 @@
   before(:all) do
     require 'will_paginate'
     MeiliSearch::Rails.configuration = {
-      meilisearch_host: ENV.fetch('MEILISEARCH_HOST', 'http://127.0.0.1:7700'),
+      meilisearch_url: ENV.fetch('MEILISEARCH_URL', 'http://127.0.0.1:7700'),
```
Comment (brunoocasali) [inline spec/integration_spec.rb:649]:
```suggestion
      meilisearch_url: ENV.fetch('MEILISEARCH_HOST', 'http://127.0.0.1:7700'),
```

File: spec/integration_spec.rb [hunk 6]
```diff
@@ -589,7 +589,7 @@
 describe 'Kaminari' do
   before(:all) do
     require 'kaminari'
-    MeiliSearch::Rails.configuration = { meilisearch_host: ENV.fetch('MEILISEARCH_HOST', 'http://127.0.0.1:7700'),
+    MeiliSearch::Rails.configuration = { meilisearch_url: ENV.fetch('MEILISEARCH_URL', 'http://127.0.0.1:7700'),
```
Comment (brunoocasali) [inline spec/integration_spec.rb:592]:
```suggestion
    MeiliSearch::Rails.configuration = { meilisearch_url: ENV.fetch('MEILISEARCH_HOST', 'http://127.0.0.1:7700'),
```

File: playground/config/initializers/meilisearch.rb [hunk 7]
```diff
@@ -1,5 +1,5 @@
 MeiliSearch::Rails.configuration = {
-    meilisearch_host: ENV.fetch('MEILISEARCH_HOST', 'http://localhost:7700'),
+    meilisearch_url: ENV.fetch('MEILISEARCH_URL', 'http://localhost:7700'),
```
Comment (brunoocasali) [inline playground/config/initializers/meilisearch.rb:2]:
```suggestion
    meilisearch_url: ENV.fetch('MEILISEARCH_HOST', 'http://localhost:7700'),
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -58,7 +58,7 @@ bundle install
 Optionally, set the credentials of the Meilisearch instance as environment variables. The defaults are:
 
 ```bash
-MEILISEARCH_HOST="http://127.0.0.1:7700"
+MEILISEARCH_URL="http://127.0.0.1:7700"
```
Comment (brunoocasali) [inline CONTRIBUTING.md:61]:
```suggestion
MEILISEARCH_HOST="http://127.0.0.1:7700"
```

**Review by brunoocasali — CHANGES_REQUESTED** [review]
Sorry if I was not so clear last comment, but you will also have to change in the code where we do explicitly references of the `MEILISEARCH_HOST`, otherwise the docker configuration will not work :)

**Comment by AdoPi** [comment]
Oops sorry I didn't understand, I'm going to change all the occurences

**Review by brunoocasali — APPROVED** [review]
Thanks a lot for your patience and your contribution! ❤️ 

bors merge

**Comment by brunoocasali** [comment]
_This message is sent automatically_

Thanks again for contributing to Meilisearch :heart:
If you are participating in [Hacktoberfest](https://blog.meilisearch.com/contribute-hacktoberfest-2022/), and you would like to receive some gift from Meilisearch too, please complete [this form](https://meilisearch.typeform.com/to/idt5qugY).

**Pull request merged**